aws-datastore/src/main/java/com/amplifyframework/datastore/storage/LocalStorageAdapter.java

```diff
@@ -186,4 +186,14 @@ Cancelable observe(
      * @throws DataStoreException if something goes wrong during terminate
      */
     void terminate() throws DataStoreException;
+
+    /**
+     * Each implementation of this adapter interface will have its own
+     * interpretation of what clear means. At a high level, the intent is to destroy
```

Comment:
1.  Remove "At a high level"
2.  In iOS, we are not making any guarantees on in-memory objects...  We still have some objects still in memory (that have read in configuration) so that customers  do not need to call configure() on amplify again if they want to use Datastore.
3.  This comment seems to be duplicated from the "*behavior.java" class... I'm not familiar with your code base, but my preference would be to only have it at the interface level.

Reply:
I've updated this. We have a little bit of duplication because we have an interface for the plugin as well as an interface for the storage adapter.